controllers/request.js

```diff
@@ -1,15 +1,37 @@
 import models from '../models';
 
-const { Request } = models;
+const { User, Request } = models;
 
 export default {
+  async approveRequest(req, res) {
+    const { params: { id }, decoded } = req;
+    const foundUser = await User
+      .findOne({ where: { id: decoded.id } });
+    await foundUser
+      .update({
+        isMentor: true
+      });
+
+    const foundRequest = await Request
+      .findOne({ where: { id } });
+
+    await foundRequest
+      .update({
+        status: 'approved'
+      });
+    return res.sendStatus(200);
+  },
   async requestToBeMentor(req, res) {
     const request = 'Request to be a mentor';
-    await Request.create({
+    const response = await Request.create({
       userId: req.decoded.id,
       request,
       status: 'pending'
     });
-    return res.send({ message: 'Your request to be a mentor has been sent' });
+    return res.send({
+      message: 'Your request to be a mentor has been sent',
+      id: response.id
+    });
```

Comment:
Why are you sending the id of the request?

Reply:
Good question. Why are you sending the `id`?

Reply:
Am sending the Id so I can get it from the response when testing, and also it will be sent as part of the email.

Reply:
Ok